qutip/tests/core/test_operators.py

```diff
@@ -342,3 +344,24 @@ def test_qzero_like(dims, superrep, dtype):
     opevo = qutip.QobjEvo(op)
     new = qutip.qzero_like(op)
     assert new == expected
+
+
+@pytest.mark.parametrize('N', [2, 3, 4, 5])
+def test_fcreate_fdestroy(N):
+    identity = qutip.tensor([*([qutip.identity(2)]*(N))])
+    zero_tensor = 0*I
+    for size_0 in range(N):
```

Comment:
Do you mean `site_0` instead of `size_0`?

Reply:
yes, thanks